scripts/startup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Startup script to execute when bootstrapping a new forensic VM.
+# The script will install forensics packages to perform analysis.
+
+max_retry=10
+
+gift_ppa_track='stable'
+
+# Packages to install
+packages=(
+  binutils
+  docker-explorer-tools
+  htop
+  jq
+  libbde-tools
+  libfsapfs-tools
+  libfvde-tools
+  ncdu
+  plaso-tools
+  sleuthkit
+  upx-ucl
+)
+
+err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
+}
+
+install_packages() {
+  add-apt-repository -y -u ppa:gift/${gift_ppa_track}
+  for package in ${packages[@]}; do
+    apt -y install ${package}
```

Comment:
It would maybe be faster if we installed them all in one go?

Reply:
Done. To address your overall comment I've also added a new argument to the `GetOrCreateAnalysisVm` to specify a particular list of packages to install.